src/components/organisms/top/top.scss

```
@@ -0,0 +1,85 @@
+@use "../../variables" as *;
+@use "../../molecules/message/message.scss" as *;
+
+.top {
+  margin-bottom: 30px;
+  .img-group {
+    position: relative;
+    background-image: url(../../../img/top.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    .literal {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      p {
+        font-family: $fontfamily-mincho;
+        color: $white;
+        writing-mode: vertical-rl;
+      }
+    }
+  }
+  .message {
+    background-color: $black;
+    h2 {
+      color: $white;
+    }
+    p {
+      color: $white;
+    }
+  }
```

```suggestion
  .message {
    background-color: $black;
    img: $white;
  }
```
これでもいいかもしれません。（message.scssの書き方によってはこれではダメかも）

---

うまくいきませんでした